worker/src/RTC/NackGenerator.cpp

```diff
@@ -68,22 +68,20 @@ namespace RTC
 		// or a retransmitted packet.
 		if (SeqManager<uint16_t>::IsSeqLowerThan(seq, this->lastSeq))
 		{
-			auto it = this->nackList.find(seq);
-
 			// It was a nacked packet.
-			if (it != this->nackList.end())
+			if (this->nackList.erase(seq) != 0u)
 			{
 				MS_DEBUG_DEV(
 				  "NACKed packet received [ssrc:%" PRIu32 ", seq:%" PRIu16 ", recovered:%s]",
 				  packet->GetSsrc(),
 				  packet->GetSequenceNumber(),
 				  isRecovered ? "true" : "false");
 
-				auto retries = it->second.retries;
-
-				this->nackList.erase(it);
+				// NOTE: Even if we didn't send yet the NACK requesting this packet (this
+				// is, if nackInfo.retries == 0), we accept this packet and let it go
+				// through because we would request it later anyway.
```

Comment:
Little rewording which makes it more clear IMO.

```suggestion
				// NOTE: Accept the packet since it was in the `nackList`, regardless
				// the NACK requesting this packet was not sent yet (this is, if nackInfo.retries == 0)
				// because we would request it later anyway.
```